src/govuk/components/back-link/_index.scss

```diff
@@ -1,10 +1,10 @@
 @include govuk-exports("govuk/component/back-link") {
 
   // Size of chevron (excluding border)
-  $chevron-size: 7px;
+  $chevron-size: .4375em;
```

Comment:
This looks brilliant 🙌 

Should we use `$govuk-root-font-size` just in case users have overridden the 16px root size?

Clearer that we're expressing 7px as ems too

```suggestion
  $chevron-size: govuk-em(7px, $govuk-root-font-size)
```

Reply:
Clarifying how the em values were created is a good shout, but I'm not sure that `$govuk-root-font-size` is the right thing to derive it from.

The em values here are with respect to the font-size being defined by the mixin on L13, which on browsers that don't support rem is *always* 16px, regardless of the value of `$govuk-root-font-size`. In that situation, a `$govuk-root-font-size` that is larger than 16px would result in an arrow that is too large relative to the link text.

However, in terms of browsers we still support that's pretty much only an IE 8 problem, which doesn't even use this graphical chevron, so this is probably a safe enough route to take.

Reply:
No worries, it looked like the [L13](https://github.com/alphagov/govuk-frontend/blob/kg-back-link-text-zoom/src/govuk/components/back-link/_index.scss#L13) mixin calls in to `govuk-px-to-rem()`

Wanted to make sure the **Back link** text didn't scale up/down with `$govuk-root-font-size`—but not the arrow

Still great to see though 😊

Reply:
In testing this out, I've noticed that at no point is `$govuk-root-font-size` actually applied to the root `<html>` element of the page. 

This means that users must manually set a matching `font-size` on `<html>` if they change `$govuk-root-font-size`, otherwise the resulting text will be sized incorrectly, demonstrated in the middle option in the table below, which is supposed to be the rem equivalent of 16px.

|`$govuk-root-font-size: 16px`<br>`html { font-size: 16px }`|`$govuk-root-font-size: 28px`<br>`html { font-size: 16px }`|`$govuk-root-font-size: 28px`<br>`html { font-size: 28px }`|
|:-:|:-:|:-:|
|<img width="135" alt="Screenshot 2022-11-10 at 15 55 57" src="https://user-images.githubusercontent.com/1253214/201143452-f1636f2d-9a18-492a-9b64-d921c20c281b.png">|<img width="135" alt="Screenshot 2022-11-10 at 15 56 06" src="https://user-images.githubusercontent.com/1253214/201143479-e3b84394-3983-4cf6-9590-8e3b64b09b42.png">|<img width="135" alt="Screenshot 2022-11-10 at 15 56 22" src="https://user-images.githubusercontent.com/1253214/201143530-a2fd9fdd-0404-49ad-89d6-b014b1dc96cf.png">|

Not really related to this PR, but also something that doesn't seem correct to my eyes. Is there any background to this, @36degrees?

Reply:
Have you seen the [Sass docs for `$govuk-root-font-size`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-root-font-size) already?

From memory, this is working as designed – we intentionally don't set the font size on the `html` element and rely on the browser default (which is nearly always 16px), but if the user _has_ for some reason done so (and set it to something other than 16px) then they might need to adjust `$govuk-root-font-size`.

Reply:
Is the root font-size only meant to be used for computation in Sass? Setting it (in px or other absolute units) on the `html` or `body` would mess with people configuring custom font-size in their browsers settings I believe 🤔

Reply:
@36degrees: I hadn't. I suppose it makes more sense if it's more of a compatibility thing than a stylistic one (i.e. a user setting it to 19px because they want everything to be 20% larger than the 16px default).

@romaricpascal: It seems like it is only used for Sass computations, yeah. AFAIK the accessibility issues with setting absolute font-size values on `html` and `body` is confined to older browsers by now—newer ones will override or zoom them anyway.

Reply:
Effectively the setting is 'Tell GOV.UK Frontend what the root font size _is_' rather than 'Tell GOV.UK Frontend what you want the root font size to be', if that makes sense at all.